crates/schema/src/def/validate/v9_old.rs

```diff
@@ -0,0 +1,921 @@
+use itertools::Itertools;
```

Comment:
This file will be removed before this is undrafted, just here while I'm porting it.